Fuzzlyn/Types/PrimitiveType.cs

```diff
@@ -92,6 +92,9 @@ static PrimitiveType()
                 AllowedAdditionalAssignments = intAssigns,
                 GenRandomLiteral = rng => CreateLiteralWithSuffix<ulong>(rng.NextUInt64(), null, "UL", (val, str) => Literal(val, str)),
                 GenTypicalLiteral = literal => CreateLiteralWithSuffix<ulong>(FromTypicalLiteral<ulong>(literal), null, "UL", (val, str) => Literal(val, str)),
+                GenTypicalLiteralLoopBounds = literal => (
+                    CreateLiteralWithSuffix<ulong>(FromTypicalLiteralLoopLowerBound<ulong>(literal), null, "UL", (val, str) => Literal(val, str)),
+                    CreateLiteralWithSuffix<ulong>(FromTypicalLiteralLoopUpperBound<ulong>(literal), null, "UL", (val, str) => Literal(val, str))),
```

Comment:
Can it just return one value if it's just returning the same thing twice?

Reply:
It returns a "Lower Bound" number that is essentially arbitrary, and an "Upper Bound" number that is (currently) the lower bound number plus 2.

Reply:
Ah, indeed, I missed that there were two variants of `FromTypicalLiteralLoop*Bound`. Now things make more sense.